nise/__init__.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "3.1.0"
+__version__ = "3.1.1"
```

Comment:
We ought to make this 4.0 since this is a pretty big change. 
```suggestion
__version__ = "4.0.0"
```